test/quantization/jit/test_ondevice_quantization.py

```diff
@@ -467,6 +468,66 @@ def _check_serialization_deserialization(self, model):
         self.assertTrue(torch.allclose(ref_output, output))
 
 
+    def _check_device_side_api(self, model):
+        model.eval()
+        inputs = model.get_example_inputs()
+        ref_m = torch.jit.script(model)
+        torch._C._jit_pass_inline(ref_m.graph)
+        qconfig_dict = {"" : default_dynamic_qconfig}
+        ref_m = prepare_dynamic_jit(ref_m, qconfig_dict)
+        ref_m = convert_dynamic_jit(ref_m)
+        buffer = io.BytesIO()
+        torch.jit.save(ref_m, buffer)
+        buffer.seek(0)
+        ref_m = torch.jit.load(buffer)
+        ref_output = ref_m(*inputs)
+
+        # check for lite interpreter
+        m = OnDevicePTQUtils.ptq_dynamic_quantize(model, qconfig_dict)
```

Comment:
Dumb question: What's the difference between `m` and `ref_m`? Any reason we need two interfaces? Is it because that there are additional functions (`quantized_forward`, `quantize_forward`, `observe_forward`, `reset_observes_forward`)?